cypress/tsconfig.json

```diff
@@ -3,9 +3,10 @@
   "extends": "../tsconfig.json",
   "compilerOptions": {
     "noEmit": true,
+    "isolatedModules": false,
```

Comment:
This was the option requiring the `export {}` at the bottom of the Cypress tests. This option is useful in the main codebase (where each file represents a module), but not in Cypress tests, so I've disabled it here.